slack_pull_reminder.py

```diff
@@ -43,14 +60,30 @@ def is_valid_title(title):
     return True
 
 
+def get_combined_status(pull_request):
+    """
+    A fun hack :/ to get the combined status of a pull request. github3.py doesn't support this atm, so hack it in
```

Comment:
This seems fine to me! Maybe add a TODO comment with the link to your github3.py pull request? 
https://github.com/sigmavirus24/github3.py/pull/897

Reply:
Good Sleuthing lol. Yeah I think I'll do that.

Reply:
I've replaced the way we get the build status and made sure to link to the pr.